src/components/editor/property/NestedProperty.jsx

```diff
@@ -6,21 +6,23 @@ import NestedPropertyHeader from './NestedPropertyHeader'
 import PropertyComponent from './PropertyComponent'
 import { selectNormProperty } from 'selectors/resources'
 import { connect } from 'react-redux'
+import shortid from 'shortid'
 import useNavigableComponent from 'hooks/useNavigableComponent'
 import { selectPropertyTemplate } from 'selectors/templates'
 
 const NestedProperty = (props) => {
   const [navEl, navClickHandler] = useNavigableComponent(props.property.rootSubjectKey, props.property.rootPropertyKey, props.property.key)
+  const propertyLabelId = `labelled-by-${shortid.generate()}`
```

Comment:
i had initially used `labeled-by-${shortid.generate()}`, and a cursory web search indicates either spelling is acceptable.  but i ended up switching to this spelling, because it's what my browser's accessibility debugger used, and consistency seemed desirable.